lib/widgets/app.dart

```diff
@@ -113,49 +115,64 @@ class ZulipApp extends StatelessWidget {
     );
 
     return GlobalStoreWidget(
-      child: Builder(builder: (context) {
-        final globalStore = GlobalStoreWidget.of(context);
-        // TODO(#524) choose initial account as last one used
-        final initialAccountId = globalStore.accounts.firstOrNull?.id;
-        return MaterialApp(
-          title: 'Zulip',
-          localizationsDelegates: ZulipLocalizations.localizationsDelegates,
-          supportedLocales: ZulipLocalizations.supportedLocales,
-          theme: theme,
-
-          navigatorKey: navigatorKey,
-          navigatorObservers: navigatorObservers ?? const [],
-          builder: (BuildContext context, Widget? child) {
-            if (!ready.value) {
-              SchedulerBinding.instance.addPostFrameCallback(
-                (_) => _declareReady());
-            }
-            GlobalLocalizations.zulipLocalizations = ZulipLocalizations.of(context);
-            return child!;
-          },
-
-          // We use onGenerateInitialRoutes for the real work of specifying the
-          // initial nav state.  To do that we need [MaterialApp] to decide to
-          // build a [Navigator]... which means specifying either `home`, `routes`,
-          // `onGenerateRoute`, or `onUnknownRoute`.  Make it `onGenerateRoute`.
-          // It never actually gets called, though: `onGenerateInitialRoutes`
-          // handles startup, and then we always push whole routes with methods
-          // like [Navigator.push], never mere names as with [Navigator.pushNamed].
-          onGenerateRoute: (_) => null,
-
-          onGenerateInitialRoutes: (_) {
-            return [
-              MaterialWidgetRoute(page: const ChooseAccountPage()),
-              if (initialAccountId != null) ...[
-                HomePage.buildRoute(accountId: initialAccountId),
-                InboxPage.buildRoute(accountId: initialAccountId),
-              ],
-            ];
-          });
+        child: Builder(builder: (context) {
```

Comment:
Update your code editors spacing to match the repo configuration. check the [ReadMe File](https://github.com/zulip/zulip-flutter?tab=readme-ov-file#code-formatting)